clients/src/main/java/org/apache/kafka/clients/admin/AlterConfigsOptions.java

```diff
@@ -31,6 +31,17 @@
 
     private boolean validateOnly = false;
 
+    /**
+     * Set the request timeout in milliseconds for this operation or {@code null} if the default request timeout for the
+     * AdminClient should be used.
+     *
+     * This method is retained to keep binary compatibility with 0.11
```

Comment:
We should probably move this to a plain comment instead of being part of the Javadoc. For users, this doesn't matter, it's just something for Kafka developers.

Reply:
@ijuma Good point. I have updated the patch as suggestd.